From: Julian <juliannfairfax@protonmail.com>
Date: Sat, 4 Jan 2025 21:21:08 +0000
Subject: arm64: dts: allwinner: pinetab: Set audio card name

Add the "PineTab" name to the sound card: this will make
upstreaming an ALSA UCM config easier as we can use a unique name.

It also avoids an issue where the default card name is truncated.

Signed-off-by: Arnaud Ferraris <arnaud.ferraris@collabora.com>
[Samuel: Split out change, updated commit message]
Signed-off-by: Samuel Holland <samuel@sholland.org>
Signed-off-by: Maxime Ripard <maxime@cerno.tech>
Signed-off-by: Julian <juliannfairfax@protonmail.com>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts
index 05194ac..94732e6 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts
@@ -508,6 +508,7 @@ hdmi_out_con: endpoint {
 
 &sound {
 	status = "okay";
+	simple-audio-card,name = "PineTab";
 	simple-audio-card,aux-devs = <&codec_analog>, <&speaker_amp>;
 	simple-audio-card,widgets = "Microphone", "Internal Microphone Left",
 				    "Microphone", "Internal Microphone Right",
